experiments/AtmosLES/convective_bl_les.jl

```diff
@@ -68,12 +68,7 @@ function main()
     dgn_config = config_diagnostics(driver_config)
 
     cbtmarfilter = GenericCallbacks.EveryXSimulationSteps(1) do
-        Filters.apply!(
-            solver_config.Q,
-            ("moisture.ρq_tot",),
-            solver_config.dg.grid,
-            TMARFilter(),
-        )
+        Filters.apply!(solver_config.Q, (), solver_config.dg.grid, TMARFilter())
```

Comment:
Wait, why are we calling this with an empty tuple? We should remove it if filtering is unnecessary.